CruCentralCoast/Database/Event.swift

```diff
@@ -12,26 +12,44 @@ import Firebase
 class Event: NSObject, DatabaseObject {
     var title: String
     var startDate: Date
-    var endDate: Date?
+    var endDate: Date
     var summary: String
-    var location: String?
+    var locationDict: NSDictionary
```

Comment:
leaving this for now until we get your code @tydahlwave code working with all of the events. You mentioned over the phone this may not be necessary for now